lib/generators/shopify_app/install/templates/shopify_app.rb.tt

```diff
@@ -4,6 +4,8 @@ ShopifyApp.configure do |config|
   config.scope = "<%= @scope %>" # Consult this page for more scope options:
                                   # https://help.shopify.com/en/api/getting-started/authentication/oauth/scopes
   config.embedded_app = <%= embedded_app? %>
+  config.new_embedded_auth_strategy = <%= embedded_app? %>
```

Comment:
Yea I thought about enabling it, but I thought it might be confusing  for apps that _aren't_ using online tokens.... 
And Paulo brings up a good point, I'm not sure why we have this check anyways, it should just not get a new token automatically if there are no online tokens or if online token doesn't have an expiry date column, and wouldn't need a config for it?

Reply:
And I just found this change to the template -- https://github.com/Shopify/shopify-app-template-ruby/pull/24

Looks offline tokens are preferred instead of using online token?